app/components/Application/ApplicationDecision.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Alert} from 'react-bootstrap';
+
+interface Props {
+  actionRequired: Boolean;
```

Comment:
```suggestion
  actionRequired: boolean;
```
using the boolean type instead of the class is preffered